.github/CODEOWNERS

```diff
@@ -13,7 +13,7 @@
 /sdk/core/                                      @pakrym @KrzysztofCwalina
 
 # Extensions
-/sdk/extensions/                                @parkym
+/sdk/extensions/                                @pakrym
```

Comment:
We probably want to apply the `Extensions` label here as well.

Reply:
Sorry, I was too quick to merge!